cmake/std/atdm/common/toss3/environment_new.sh

```diff
@@ -0,0 +1,201 @@
+################################################################################
+#
+# Set up env on toss3 (chama and serrano) for ATMD builds of Trilinos
+#
+# This source script gets the settings from the ATDM_CONFIG_BUILD_NAME var.
+#
+################################################################################
+
+if [ "$ATDM_CONFIG_COMPILER" == "DEFAULT" ] ; then
+  export ATDM_CONFIG_COMPILER=INTEL
+fi
+
+echo "Using toss3 compiler stack $ATDM_CONFIG_COMPILER to build $ATDM_CONFIG_BUILD_TYPE code with Kokkos node type $ATDM_CONFIG_NODE_TYPE"
+
+export ATDM_CONFIG_ENABLE_SPARC_SETTINGS=ON
+export ATDM_CONFIG_USE_NINJA=ON
+export ATDM_CONFIG_BUILD_COUNT=8
+# export ATDM_CONFIG_CMAKE_JOB_POOL_LINK=2
+# NOTE: Above, currently setting CMAKE_JOB_POOL_LINK results in a build
+# failures with Ninja.  See https://gitlab.kitware.com/snl/project-1/issues/60
+
+# We do this twice since sems modules are wacked and we get errors to the screen on a purge
+# The second purge will catch any real errors with purging ...
+module purge &> /dev/null
+module purge
+. /projects/sems/modulefiles/utils/sems-modules-init.sh
+module load sems-env
+module load atdm-env
+module load atdm-ninja_fortran/1.7.2
+
+if [ "$ATDM_CONFIG_NODE_TYPE" == "OPENMP" ] ; then
+  export ATDM_CONFIG_CTEST_PARALLEL_LEVEL=8
+  export OMP_NUM_THREADS=2
+else
+  export ATDM_CONFIG_CTEST_PARALLEL_LEVEL=16
+fi
+
+if [ "$ATDM_CONFIG_COMPILER" == "INTEL" ]; then
```

Comment:
As I discussed with Evan, we likely need to add a [custom_builds.sh](https://github.com/trilinos/Trilinos/blob/develop/cmake/ctest/drivers/atdm/README.md#how-to-add-a-new-system) file because we need to support Intel 18 and Intel 19 builds.  See the updated list of subtasks in [ATDV-212](https://sems-atlassian-srn.sandia.gov/browse/ATDV-212).

Reply:
Added in 66bdf2a.